lib/compiler/src/sys_core_fold.erl

```diff
@@ -1405,7 +1406,7 @@ is_subst(_) -> false.
 %%
 %%  We use the variable name as key so as not have problems with
 %%  annotations.  When adding a new substitute we fold substitute
-%%  chains so we never have to search more than once.  Use orddict so
+%%  chains so we never have to search more than once.  Use map so
```

Comment:
`"Use map so we know the format."` does not make much sense. I suggest you delete the sentence.